lckernel/cad/math/intersectionhandler.cpp

```diff
@@ -289,3 +289,66 @@ void Intersection::BezCir(const geo::Circle C, const geo::Area c_area, const geo
     BezCir(C, c_area, m_area, b2split[1], ret);
 
 }
+
+std::vector<geo::Coordinate> Intersection::SplineLine(geo::Spline B, const geo::Vector& V) {
+    std::vector<geo::Coordinate> ret;
+    auto beziers = B.beziers();
+    for(const auto & bezier : beziers) {
+        auto vecret = BezierLine(bezier, V);
```

Comment:
You are correct about the 'get' prefix, that's my java brain thinking and as you are saying, we don't use that in LibreCAD.
That said, in LibreCAD we always started all method names and variable names with lower case and class names with a uppercase letter this just read's better and this is very common in the industry.

For example:
BezierLine.getPoint();

Is this a static method getPoint(); on a class or is this a instance method call?

Similar as shown before, if the method name start's with a upper case letter it reads as if it's object creation, not as method calling.

So, method names are start with lower case, class names start with a Uppercase letter, constants are all uppercase.

A very long time ago I created this http://wiki.librecad.org/index.php/LibreCAD_3_-_LibreCAD_3_-_code_style I know it's not great, but at the bottom it's mentioned.